test/sqllogictest/transform/predicate_pushdown.slt

```diff
@@ -154,8 +153,7 @@ SELECT * FROM t1, t2 WHERE t1.f1 = t2.f1 AND t1.f2 = t2.f2 AND t1.f1 + t2.f2 = t
 | | implementation = DeltaQuery
 | |   delta %0 %1.(#0, #1)
 | |   delta %1 %0.(#0, #1)
-| | demand = (#0, #1)
-| Filter ((#0 + #1) = (#0 + #1))
+| Filter ((#0 + #1) = (#0 + #1)), ((#2 + #3) = (#2 + #3))
```

Comment:
More obvious example of the absence of a post-join `Project` to unify equated columns, I think.